If the changes to sys/init_sysent.c succeeded, then this should be fine.
If not, you should check that the system call numbers added coincide with
those in sys/init_sysent.c.
***************
*** 154,159
  /* 152 thru 165 reserved */
  #define SYS_setprocesshost 166
  #define SYS_getprocesshost 167
  
  /*
   * BTL system call header table.

--- 154,162 -----
  /* 152 thru 165 reserved */
  #define SYS_setprocesshost 166
  #define SYS_getprocesshost 167
+ #define SYS_remoteon	168
+ #define SYS_remoteoff	169
+ #define SYS_remotename	170
  
  /*
   * BTL system call header table.
